src/ft2/_ft2.cpp

```diff
@@ -0,0 +1,635 @@
+#include "_ft2.h"
+#include "_layout.h"
+
+namespace matplotlib::ft2 {
+
+Face::Face(std::string const& path, double hinting_factor) :
+  ptr{
+    [&]() -> std::shared_ptr<FT_FaceRec_> {
+      auto c_path = const_cast<char*>(path.c_str());
```

Comment:
Ain't we will get a problem with unicode paths? Currently file handling stuff happens in python.

Reply:
Could (should) be rewritten to pass a stream object in.